### ISSUE TITLE
A11Y: Template images are not labeled

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#377]
**ENVIRONMENT:**

OS (number): N (7.1.2)
Build number: N2G47H
Form factor (Smartphone / tablet):  Nexus 6P
Browser: Chrome Dev 59.0.3056.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)))
2. Try to Linearly navigate through different template images present under Page Templates heading.

**ACTUAL RESULTS:**

Template Images are read out as “Template 3_2x graphic”, “Template 1_2x graphic” etc.

**EXPECTED RESULTS:**

If images are purely decorative and doesn’t convey any info, then alt tag should be “null”, so screen reader can skip them. If Images provide useful information then, appropriate label should be used for the images along with description (Note: Description should be used whenever required.)

**ATTACHMENT(S):**

<img width="1440" alt="unlabeledtemplateimages" src="https://cloud.githubusercontent.com/assets/26986019/24814455/8103990c-1b86-11e7-9e4d-00f059e9f41f.png">

**Comment by spotukuchi** [comment]
This issue is reproducible in the below environment:

ENVIRONMENT:
Form factor : iPAD
Model : MC705LL/A
Version : 9.3.5(13G36)
Browser : Safari

PRE-REQUISITE:
Turn 'VoiceOver' ON. ( Settings->General->Accessibility->VoiceOver->ON )

STEPS TO REPRODUCE:
1. Navigate to the URL: https://ampstart.com
2. Try to Linear navigate through different template images present under Page Templates heading.

ACTUAL RESULTS:
Template Images are read out as “Nasa 43657 JPG” etc.

EXPECTED RESULTS:
If images are purely decorative and doesn’t convey any info, then alt tag should be “null”, so screen reader can skip them. If Images provide useful information then, appropriate label should be used for the images along with description (Note: Description should be used whenever required.)

Please see the video in the link below for reference.

https://drive.google.com/corp/drive/folders/0BwgKOOZP99EuUzVWT25TVzl2S00

**Comment by torch2424** [comment]
This was closed by #458
